README.adoc

```diff
@@ -588,13 +588,20 @@ FAQ.md                  100% 1831     1.8KB/s   00:00
 GRAMMAR.md              100% 1666     1.6KB/s   00:00
 ```
 
-A variadic parameter with a default argument will accept zero or more arguments:
+Variadic parameters prefixed with `*` accept _zero or more_ arguments. They expand to a string containing those arguments separated by spaces, or an empty string if no arguments are present:
 
 ```make
-commit MESSAGE +FLAGS='':
+commit MESSAGE *FLAGS:
   git commit {{FLAGS}} -m "{{MESSAGE}}"
 ```
 
+Variadic parameters prefixed by `+` can be assigned default values. These are overridden by arguments passed on the command line:
```

Comment:
Nothing that makes it problematic per se, but if a star variadic has a default it can never be called with zero instantiations of the parameter. To me that sort of defeats the purpose of something that is "zero or more", but I can certainly appreciate that for consistency it might be better to allow this.